project/src/components/review-list/review-list.tsx

```diff
@@ -0,0 +1,17 @@
+import ReviewItem from './review-item/review-item';
+
+import CommentType from '../../types/commentType';
+
+type ReviewListProps = {
+  reviewsData: CommentType[],
+};
+
+function ReviewList({ reviewsData }: ReviewListProps) {
+  return (
+    <ul className="reviews__list">
+      {reviewsData.map((review) => <ReviewItem key={review.id} reviewData={review} />)};
```

Comment:
```suggestion
      {reviewsData.map((review) => <ReviewItem key={review.id} reviewData={review} />)}
```
<img width="635" alt="image" src="https://user-images.githubusercontent.com/15823482/158433923-3b32e820-5648-4c60-8b21-bffb44bbd977.png">